### PR TITLE
Explain remix as codeviewer

### PR DESCRIPTION
**Pull request by Aniket-Engg**
documented https://github.com/ethereum/remix-project/pull/2536

### CODE REVIEW

File: docs/remix_as_code_viewer.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Remix as code viewer
+=============================
+
+Through Etherscan
+------------------
+
+If there is a contract verified on Etherscan and someone wants to have a look on it in an IDE, mostly for a mutiple part contract verification, Remix provides a quick way to load the contract.
```
Comment (ryestew) [inline docs/remix_as_code_viewer.md:7]: I got a bit lost in your first sentence.  What do you think of this edit:
Verified contracts on Etherscan can be viewed in Remix by making a simple change to the URL.

File: docs/remix_as_code_viewer.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Usually, contract is shown on Etherscan on a URL like this:
```
Comment (ryestew) [inline docs/remix_as_code_viewer.md:9]: This sentence could be:
A typical Etherscan URL looks like this:
or
Usually, a contract is shown on Etherscan with a URL like this:

File: docs/remix_as_code_viewer.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+`https://etherscan.io/address/0xdac17f958d2ee523a2206206994597c13d831ec7`
+
+In URL, just update `etherscan.io` with `remix.ethereum.org`
```
Comment (ryestew) [inline docs/remix_as_code_viewer.md:13]: In the URL, update ... or 
In the URL, just update 

(as a note, the use of "just" or "simply" stylistically annoys me when I read it - well it annoys me when the directions say it is simple but I can't do it.  But use it or not - I wouldn't ask you to change that)...actually on second thought- "just" doesn't really bother me.  Simply does. But you didn't use that here.

File: docs/remix_as_code_viewer.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+`https://remix.ethereum.org/address/0xdac17f958d2ee523a2206206994597c13d831ec7`
+
+and reload. It will fetch the contracts verified on Ethereum mainnnet and on other testnets (Ropsten, Rinkeby, Kovan & Goerli).
+
+![](images/a-code-viewer-etherscan.png)
+
+This works for Etherscan testnet URLs like `https://ropsten.etherscan.io` etc.
```
Comment (ryestew) [inline docs/remix_as_code_viewer.md:21]: this line is basically a repeat of line 17.

File: docs/remix_as_code_viewer.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+Loaded contracts will be stored under `etherscan-code-sample` workspace.
```
Comment (ryestew) [inline docs/remix_as_code_viewer.md:23]: Loaded contracts will be stored in the `etherscan-code-sample` workspace.

File: docs/remix_as_code_viewer.md (hunk 1 above)
Comment (ryestew) [reply docs/remix_as_code_viewer.md:7]: ahh but here I'm using "simple"...(see below)

File: docs/remix_as_code_viewer.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+![](images/a-code-viewer-etherscan.png)
+
+This works for Etherscan testnet URLs like `https://ropsten.etherscan.io` etc.
+
+Loaded contracts will be stored under `etherscan-code-sample` workspace. 
+
+Through GitHub
+------------------
+
+If a .sol file is opened in Github and someone wants to just compile it or try some changes with it, file can be loaded on Remix similarly. For a file with URL like:
```
Comment (ryestew) [inline docs/remix_as_code_viewer.md:28]: Solidity files in Github can be loaded on Remix with a similar switch. For a file with URL like:

**Review by ryestew — CHANGES_REQUESTED** [review]

File: docs/remix_as_code_viewer.md (hunk 1 above)
Comment (Aniket-Engg) [reply docs/remix_as_code_viewer.md:7]: updated

File: docs/remix_as_code_viewer.md (hunk 2 above)
Comment (Aniket-Engg) [reply docs/remix_as_code_viewer.md:9]: updated

File: docs/remix_as_code_viewer.md (hunk 6 above)
Comment (ryestew) [reply docs/remix_as_code_viewer.md:28]: If you don't like the edit above, please add a "the" as in
the file can be loaded on Remix similarly.

**Review by ryestew — APPROVED** [review]
I'll approve this - but it would be good to add a "the" as specified in the comment for remix-as-code-viewer.md line 28

File: docs/remix_as_code_viewer.md (hunk 3 above)
Comment (Aniket-Engg) [reply docs/remix_as_code_viewer.md:13]: thanks for explanation. I updated it.

File: docs/remix_as_code_viewer.md (hunk 4 above)
Comment (Aniket-Engg) [reply docs/remix_as_code_viewer.md:21]: 17 one says that contracts will be loaded from different testnets and this says that you can also update testnet urls too

Comment (Aniket-Engg) [reply docs/remix_as_code_viewer.md:21]: I update to make it more clear

File: docs/remix_as_code_viewer.md (hunk 6 above)
Comment (Aniket-Engg) [reply docs/remix_as_code_viewer.md:28]: updated

**Pull request merged**